targets/esp32/main.c

```diff
@@ -51,7 +53,8 @@ static void espruinoTask(void *data) {
   jshInit();     // Initialize the hardware
   jswrap_ESP32_wifi_restore();
   jsvInit();     // Initialize the variables
-  vTaskDelay(1000 / portTICK_PERIOD_MS);
+  // not sure why this delay is needed?
+  vTaskDelay(200 / portTICK_PERIOD_MS);
```

Comment:
As far as I remember this delay already existed in first version of main.c, written by nkolban